client/src/components/Login/LoginContainer.tsx

```diff
@@ -1,51 +1,46 @@
-import React from 'react';
-import {connect} from 'react-redux';
+import React from "react";
 import Login from "./Login";
-import {setEmailText, setPasswordText, getApiData} from "../../store/login/actions";
-
-interface FormProps {
-    email: string;
-    password: string;
-    token: any;
-    setEmailText: Function;
-    setPasswordText: Function;
-    getApiData: Function;
-}
-
-interface LoginState {
-    email: string;
-    password: string;
-    token: any;
-}
-
-
-interface FormState {
-    login: LoginState;
-}
-
-class LoginContainer extends React.Component<FormProps, FormState> {
-    render() {
-        return <Login email={this.props.email}
-                      password={this.props.password}
-                      setEmailText={this.props.setEmailText}
-                      setPasswordText={this.props.setPasswordText}
-                      getApiData={this.props.getApiData}
-        />;
-    }
-}
-
-const mapStateToProps = (state: FormState) => {
-    return {
-        email: state.login.email,
-        password: state.login.password,
-        token: state.login.token
-    };
+import { reduxForm } from "redux-form";
+import { connect } from "react-redux";
+import { loginUser } from "../../store/login/actions";
+
+const validate = (user: any) => {
+  const errors: any = {};
+  const validEmail = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i
```

Comment:
the same regexp as prev? So if I would like to change it, I 'll have to change it in two places?